tests/foreman/cli/test_errata.py

```diff
@@ -198,11 +360,19 @@ def test_negative_install_by_hc_id_without_errata_info(self):
 
         @Assert: Error message thrown.
 
-        @caseautomation: notautomated
-
         """
+        with self.assertRaises(CLIReturnCodeError) as context:
+            HostCollection.erratum_install({
+                'id': self.host_collection['id'],
+                'organization-id': self.org['id'],
+            })
+        self.assertIn(
+            "Error: option '--errata' is required",
```

Comment:
Not sure if we shall check the error messages, as they may be localized. @oshtaier how do you think?